CHANGELOG.md

```diff
@@ -105,6 +105,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Metroid Prime
 
+- Fixed: Crashes in several rooms when skipping cutscenes exactly 1 second from the end
```

Comment:
i thought this was 1 frame?

Reply:
It's a frame-perfect start press 1 second before end

The fade-out takes 1 second to complete and it's when that fade ends that we have the edge case